docs/src/man/comparisons.md

```diff
@@ -1,77 +1,173 @@
+| Complex Function          | `df[['z']].agg(lambda v: np.mean(np.cos(v)))`                                | `combine(df, :z => (v -> mean(cos, skipmissing(v))))`         |
+|                           | `df['z'].mean()`                                                             | `combine(df, :z => (v -> mean(skipmissing(v))))` |
+| Transform several columns | `df.agg({'x': max, 'y': min})`                                               | `combine(df, :x => maximum, :y => minimum)`                   |
+|                           | `df[['x','y']].mean()`                                                       | `combine(df, [:x, :y] .=> mean)`                              |
+|                           | `df.filter(regex=("^x")).mean()`                                             | `combine(df, r"^x" .=> mean)`                                 |
+|                           | `df[['x', 'y']].agg([max, min])`                                             | `combine(df, ([:x, :y] .=> [maximum minimum])...)`            |
+| Multivariate function     | `df.assign(x_y_cor = np.corrcoef(df.x, df.y)[0,1])`                          | `transform(df, [:x, :y] => cor)`                              |
+| Row-wise                  | `df.assign(x_y_min = df.apply(lambda v: min(v.x, v.y), axis=1))`             | `transform(df, [:x, :y] => ByRow(min))`                       |
+|                           | `df.assign(x_y_argmax = df.apply(lambda v: df.columns[v.argmax()], axis=1))` | `transform(df, AsTable([:x,:y]) => ByRow(argmax))`            |
+| DataFrame as input        | `df.groupby('grp').head(2)`                                                  | `combine(d -> first(d, 2), groupby(df, :grp))`                |
+| DataFrame as output       | `df[['x']].agg(['max','min'])`                                               | `combine(:x => x -> (value = [minimum(x), maximum(x)],), df)` |
+
+### Joining data frames
+
+| Operation             | pandas                                                  | DataFrames.jl                           |
+|:----------------------|:--------------------------------------------------------|:----------------------------------------|
+| Inner join            | `pd.merge(df1, df2, how = 'inner', on = ['id', 'grp'])` | `innerjoin(df1, df2, on = [:id, :grp])` |
+| Outer join            | `pd.merge(df1, df2, how = 'outer', on = ['id', 'grp'])` | `outerjoin(df1, df2, on = [:id, :grp])` |
+| Left join             | `pd.merge(df1, df2, how = 'left', on = ['id', 'grp'])`  | `leftjoin(df1, df2, on = [:id, :grp])`  |
+| Right join            | `pd.merge(df1, df2, how = 'right', on = ['id', 'grp'])` | `rightjoin(df1, df2, on = [:id, :grp])` |
+| Semi join (filtering) | `df1[df1.id.isin(df2.id)]`                              | `semijoin(df1, df2, on = :id)`          |
+| Anti join (filtering) | `df1[~df1.id.isin(df2.id)]`                             | `antijoin(df1, df2, on = :id)`          |
```

Comment:
can we give an example using two keys - for consistency with the examples above?

Reply:
Let's keep it simple. I will change all the other examples to join on a single column to match.